pages/phases/certificate.md

```diff
@@ -24,23 +24,30 @@ roles: [hr, hiring-mgr]
   <div class="usa-alert__body">
     <h3 class="usa-alert__heading">Do not conduct a separate minimum qualification review</h3>
     <p class="usa-alert__text">
-      After SMEs have completed interviews and identified the applicants who are found to meet qualifications, do not conduct a separate minimum qualification review. For more information, see "<a href="{{ site.baseurl }}/about/differences/#sme-assessment">How our process is different</a>."
+      After SMEs have completed interviews and identified the applicants who are found to meet qualifications, do not conduct a separate minimum qualification review. For the duration of the hiring action, SMEs have used their expertise and technical ability to assess whether an applicant is truly qualified before being considered minimally qualified and eligible for Veterans' Preference. As the HR specialist, you should review pay settings for applicants who are current federal employees to make sure they qualify for the certificate. For more information, see "<a href="{{ site.baseurl }}/about/differences/#sme-assessment">How our process is different</a>."
```

Comment:
this wasn't the sentence i meant to add- i meant to add:

 After SMEs have completed interviews and identified the applicants who are found to meet qualifications, do not conduct a separate minimum qualification review. Throughout the hiring action, SMEs have used their expertise and technical ability to assess whether an applicant is truly qualified and as HR, you already reviewed every justification to ensure it was rooted in one of the technical qualifications.  No additional minimum qualification check is required in this process.